packages/gestalt-datepicker/src/DatePicker.css

```diff
@@ -0,0 +1,310 @@
+:root {
+  --color-darkGray: #111;
+  --color-gray: #767676;
+  --color-hover-lightGray: rgba(0, 0, 0, 0.06);
+  --color-white: #fff;
+  --borderRadius-circle: 50%;
+  --borderRadius-halfCircle-End: 0 50% 50% 0;
+  --borderRadius-halfCircle-Start: 50% 0 0 50%;
+  --day-dimensions: 40px;
+  --empty-content: "";
```

Comment:
@AlbertCarreras let's inline this, not sure why we need a separate variable

Reply:
fixed